modules/configmap-removing-ca.adoc

```diff
@@ -6,12 +6,12 @@
 [id="configmap-removing-ca_{context}"]
 = Removing certificate authorities on a {product-title} cluster
 
-You can remove certificate authorities (CA) from your cluster with the `rosa` CLI tool.
+You can remove certificate authorities (CA) from your cluster with the {product-title} (ROSA) CLI, `rosa`, tool.
```

Comment:
This sentence is pretty clunky now - "with the {product-title} (ROSA) CLI, `rosa`." is probably fine without "tool" but if you include "tool" I'd keep it together with CLI, e.g. CLI tool, `rosa`.

Reply:
Removed "tool".